### DISCUSSION TITLE
Start up text menu no longer appearing

### DISCUSSION THREAD

**Discussion by bytebeast** [goolord/alpha-nvim#243] [Q&A] [answered]
My start up screen is appearing with only my ansi image, but text menu under ansi image is no longer under it. This started happening about 1 week ago. 

If i remove everything under ~/local/nvim/{state,share} , then it recreates everything and it works for a little bit...but then disappears after a few times.

**Comment by goolord** [comment]
can you link / paste your config please?

it sounds like something may be happening with your `oldfiles`, not sure it's related to `alpha` but I don't know what your config is like so I don't know how to help

**Reply by bytebeast** [reply]
posted my config below

**Comment by bytebeast** [comment]
This the config i'm using with lazyvim:
```lua
-- omitted, no longer relevant.
```

**Reply by goolord** [reply]
```lua
      -- close Lazy and re-open when the dashboard is ready
      if vim.o.filetype == "lazy" then
        vim.cmd.close()
        vim.api.nvim_create_autocmd("User", {
          pattern = "AlphaReady",
          callback = function()
            require("lazy").show()
          end,
        })
      end
```
i have a feeling this is the problem, the detection of open buffers on startup recently became more robust

**Comment by bytebeast** [comment, marked as answer]
Found the issue. Commenting this line, fixed it.
```lua
pcall(vim.cmd.AlphaRedraw)
```
